src/clients/host_auto.rb

```diff
@@ -153,10 +155,16 @@ def check_profile_for_errors(imported_hosts)
       imported_hosts.each do |ip, hosts|
         next unless hosts.any? { |host| host.strip.empty? }
 
-        AutoInstall.issues_list.add(:invalid_value, "host", "names",
+        AutoInstall.issues_list.add(
+          ::Installation::AutoinstIssues::InvalidValue,
+          Y2Network::AutoinstProfile::HostSection.new_from_hashes(
```

Comment:
This is not the right place to check for errors (it should be part of Hosts.Import). Please, add a `# TODO` comment here and I will take care on Monday.